4-forms-validations/forms.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Form</title>
+    <style>
+      body {
+        background-color: darkgreen;
+      }
+  
+      form {
+        margin: 5px;
+        background-color: lightgreen;
+      }
+  
+      button {
+        padding: 10px;
+        color: white;
+        background-color: green;
+        font-family: monospace;
+        margin-left: 45%;
+        border-radius: 15px;
+      }
+  
+      footer {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <form action="">
+      <fieldset>
+        <legend><h1>Personal Data Form:</h1></legend>
+        <ol>
+          <li>
+            <label for="first_Name"><b>First Name:</b><abbr title="required" aria-label="required">*</abbr>
+              <input type="text" id="first_Name" placeholder="Insert your first name" required /></label>
+          </li>
+          
+          <li>
+            <label for="last_Name"
+              ><b>Last Name:</b>
+              <abbr title="required" aria-label="required">*</abbr>
+              <input
+                type="text"
+                id="last_Name"
+                placeholder="Insert your last name"
+                required
+            /></label>
+          </li>
+          
+          <li>
+            <label for="email"
+              ><b>Email:</b>
+              <abbr title="required" aria-label="required">*</abbr>
+              <input
+                type="email"
+                id="email"
+                placeholder="Insert your Email"
+                required
+            /></label>
+          </li>
+          
+          <li>
+            <label for="birth_date"
+              ><b>Write your BirthDate:</b>
+              <abbr title="required" aria-label="required">*</abbr>
+              <input
+                type="date"
+                id="birth_date"
+                placeholder="Insert your birth_date"
+                required
+            /></label>
+          </li>
+          
+          <li>
+            <label for="genre"
+              ><b>Pick your Genre:</b>
+              <abbr title="required" aria-label="required">*</abbr>
+              <select id="genre">
+                <option value="Male">Male</option>
+                <option value="Female">Female</option>
+                <option value="Other">Other</option>
+              </select>
+            </label>
+          </li>
+          
+          <li>
+            <label for="hobbies">
+              <b>Pick your Hobbies:</b>
+              <abbr title="you can choose one or more" 
+                >^</abbr
+              >
+              
+              <input type="radio" name="gaming" />
+              <label for="gaming">Playing Videogames</label>
```

Comment:
```suggestion
              <input type="radio" name="gaming" id="gaming" />
              <label for="gaming">Playing Videogames</label>
```
For the labels to be accessible, they need the id of their corresponding element.